### PR TITLE
Ui kit

### PR DESCRIPTION
**Pull request by UtopiaEH**
## Changelog
```changelog
 - NASA: Added UI kit and sass
```
  
## Change Type
- [x] `Added`: new feature - non-breaking change which adds functionality.
- [ ] `Bug fix`: non-breaking change which fixes an issue.
- [ ] `Changed`: for changes in existing functionality.
- [ ] `Deprecated`: for soon-to-be removed features.
- [ ] `Documentation` update.
- [ ] `Performance`: for performance and optimization functionality.
- [ ] `Removed`: for now removed features.
- [ ] `Security`: in case of vulnerabilities.
## Required Checks
 - [ ] Documentation
 - [ ] Unit tests
 - [x] Tested on Chrome
 - [ ] Tested on Firefox
 - [ ] Tested on IE11
 - [x] npm run eslint
 - [x] npm run prettier
## Screenshots
________________

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "baseUrl": "src",
     "paths": {
       "common": ["src/common/*"],
-      "utils": ["src/utils/*"]
+      "utils": ["src/utils/*"],
+      "ui-kit": ["src/ui-kit/*"]
```
Comment (cristian-sandu) [inline jsconfig.json]: 👍

File: jsconfig.json [hunk 2]
```diff
@@ -2,8 +2,10 @@
   "compilerOptions": {
     "baseUrl": "src",
     "paths": {
+      "include": ["src"],
```
Comment (cristian-sandu) [inline jsconfig.json]: why do we need this **include** ?

File: src/ui-kit/button.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button as AntdButton } from 'antd'
+
+const Button = ({ title }) => (
+  <span>
+    <AntdButton style={{ fontSize: '25px' }}>{title}</AntdButton>
+  </span>
+)
+Button.propTypes = {
```
Comment (cristian-sandu) [inline src/ui-kit/button.js]: this is not readable at all..put an empty line between

Comment (cristian-sandu) [reply src/ui-kit/button.js]:
```
const Button = ({ title }) => (
  <span>
    <AntdButton style={{ fontSize: '25px' }}>{title}</AntdButton>
  </span>
)

Button.propTypes = {...}
```

Comment (cristian-sandu) [reply src/ui-kit/button.js]: also you can directly import the function you like from prop-types, the last version of prop-types supports that:

**import { func } from 'prop-types'**
```
Button.propTypes = {
  func.isRequired
}
```

File: src/ui-kit/button.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  title: PropTypes.func.isRequired,
+}
+export default Button
```
Comment (cristian-sandu) [inline src/ui-kit/button.js]: add a empty line here as well...

File: src/ui-kit/button.js (hunk 3 above)
Comment (UtopiaEH) [reply src/ui-kit/button.js]: I understand what you want say, but i don't find another example: ` import { func } from 'prop-types'`

File: jsconfig.json (hunk 2 above)
Comment (UtopiaEH) [reply jsconfig.json]: i fix it

File: jsconfig.json [hunk 5]
```diff
@@ -3,7 +3,9 @@
     "baseUrl": "src",
     "paths": {
       "common": ["src/common/*"],
-      "utils": ["src/utils/*"]
+      "utils": ["src/utils/*"],
+      "ui-kit": ["src/ui-kit/*"],
+      "scss-kit": ["src/ui-kit/*"]
```
Comment (cristian-sandu) [inline jsconfig.json]: why do we need to duplicate this ?????????????????????????????????????????? they point to the same  path - src/ui-kit/*

File: src/ui-kit/button.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { string } from 'prop-types'
+import { Button as ButtonAntD } from 'antd'
+import './scss/button.scss'
+
+const DEFAULT_TITLE = 'Submit'
+const Button = ({ title }) => (
```
Comment (cristian-sandu) [inline src/ui-kit/button.js]: please put an empty line between line 6 and 7, we've already talked about this

**Pull request merged**